Makefile

```diff
@@ -26,8 +26,8 @@ $(VENDOR_CHECKS): check-%-version:
 
 
 .PHONY: clean
-clean: $(FILES)
-	rm -f -- $^
+clean:
+	rm -f -- $(FILES)
```

Comment:
This was silly. By making all the files dependencies of `clean`, `make clean` would *make* the files before deleting them. :dizzy_face: